spec/unit/knife/winrm_spec.rb

```diff
@@ -443,6 +443,22 @@
       expect { @winrm.run_with_pretty_exceptions }.to raise_error(SystemExit)
     end
 
+    context "impact of concurrency value" do
+      before(:each) do
+        allow(Chef::Knife::WinrmSession).to receive(:new).and_return(session)
+        @winrms = Chef::Knife::Winrm.new(['-C', '3', '-m', 'localhost', '-x', 'testuser', '-P', 'testpassword', '--winrm-authentication-protocol', 'basic', 'echo helloworld'])
+      end
+      it "create sessions to equal number of concurrent value if concurrent option is given" do
```

Comment:
As per the description, can we have a number check in expectation ?
Also in the upcoming test cases.